Formula/README.md

```diff
@@ -0,0 +1,39 @@
+# Create `brew` formula
+
+One cannot setup the `venv` using poetry cause [`poetry` does not allow to install stictdoc in development mode yet](https://github.com/python-poetry/poetry/issues/34).
+[Create and activate a `venv` the plain Python way](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments) instead.
+
+```
+python3 -m venv venv-strictdoc-homebrew
+source venv-strictdoc-homebrew/bin/activate
+```
+
+Install `brew` on Mac OS or Linux (on Linux into `/home/linuxbrew/.linuxbrew/bin/brew`) like [described on the website](https://brew.sh/).
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+Install `strictdoc` and the homebrew helper `poet` for generating Python formulas into the `venv`:
+
+    pip install strictdoc homebrew-pypi-poet
+
+Generate the Homebrew formula for `strictdoc` including it's dependencies and leave the `venv`:
+
+    poet -f strictdoc > Formula/strictdoc.rb
```

Comment:
Does this mean that we eventually have to connect this task to the main Pip package release cycle otherwise the formula will get outdated?

Reply:
For an automated consistent release process, yes. As homebrew is compatible with Mac OS and Linux one could automate the release in [`release.yml`](https://github.com/strictdoc-project/strictdoc/blob/main/.github/workflows/release.yml) by adding 2 jobs, one running on Mac OS `runs-on: macOS-latest` and one on Linux `runs-on: ubuntu-latest` to test if the brew package has been built successfuly and runs on both OSes. As there are no known issues specific to Linux and/or Mac OS (like for [Windows](https://github.com/strictdoc-project/strictdoc/issues/491)) it could be sufficient to add a single job to build for MacOS and expect that it works on Linux as well. However I've not used GitHub workflows (I'm a GitLab user).

Reply:
There is a release job here: https://github.com/strictdoc-project/strictdoc/blob/main/.github/workflows/release.yml. I have asked because I would prefer this process of upgrading the formula to be automated.

Could you investigate if there is an easy way to automate this by extending the existing release job or adding an extra one? In addition, your instructions for building the formula: is there an option for including these commands to the `tasks.py` file similarly like I am doing it here: https://github.com/strictdoc-project/reqif/blob/main/tasks.py#L234. Something like `invoke brew-update`?

Reply:
The steps required to release to the homebrew repositoy are explained in [*How To Open a Homebrew Pull Request*](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). One needs to [fork the formulas repo first](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md#formulae-related-pull-request) and release an initial package version. As soon the first version of a package has been accepted and released it seems to be sufficient to use the [`brew bump-formula-pr` command](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md#submit-a-new-version-of-an-existing-formula). As soon as the brew package deplioyment pipeline is properly setup there is no need to consider [follow up instructions](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md#following-up) anymore.